challenges/objects-arrays.js

```diff
@@ -98,28 +250,38 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 */
 
 const lowerCase = [];
-console.log(lowerCase); 
+zooAnimals.map(name => lowerCase.push(name.animal_name.toLowerCase()));
+console.log(lowerCase);
```

Comment:
The `.map` method returns a new array. Data should not be mutated within the body.
```js
const lowerCase = zooAnimals.map(animal => animal.animal_name.toLowerCase())
```